jobs/generation/Utilities.groovy

```diff
@@ -140,27 +173,27 @@ class Utilities {
         }
 
         def passCI = ''
-        if (project == 'corefx' && branch == 'master') {
+        if (project == 'corefx' && (branch == 'master' || branch == 'release/3.0')) {
           // Master uses Arcade for build and requires --ci option to be passed
           passCI = '--ci'
         }
 
         def authorsOpts = '/p:Authors=Tizen'
 
         if (project == 'coreclr') {
-          shell("${dockerCommand} ./build.sh cross \${config} \${targetArch} cmakeargs -DFEATURE_GDBJIT=TRUE stripSymbols -PortableBuild=false -- \${buildIdOpts} \${stableOpts} \${packageOpts} ${authorsOpts}")
+          shell("${dockerCommand} ./build.sh cross \${config} \${targetArch} cmakeargs -DFEATURE_GDBJIT=TRUE cmakeargs -DFEATURE_PREJIT=true stripSymbols -PortableBuild=false -- \${buildIdOpts} \${stableOpts} \${packageOpts} ${authorsOpts}")
         } else if (project == 'corefx') {
-          if (branch == 'master') {
+          if (branch == 'master' || branch == 'release/3.0') {
             // Build command for CoreFX has changed: see https://github.com/dotnet/corefx/pull/32798/files and https://github.com/dotnet/corefx/commit/66392f577c7852092f668876822b6385bcafbd44
 
-            shell("${dockerCommand} ./build.sh -\${config} /p:ArchGroup=\${targetArch} /p:RuntimeOS=tizen.5.0.0 /p:PortableBuild=false ${passCI} \${buildIdOpts} \${stableOpts} \${packageOpts} /p:BinPlaceNETCoreAppPackage=true /p:OverridePackageSource=https:%2F%2Ftizen.myget.org/F/dotnet-core/api/v3/index.json ${authorsOpts}")
+            shell("${dockerCommand} ./build.sh --configuration \${config} /p:ArchGroup=\${targetArch} /p:RuntimeOS=${getTizenVersion(branch)} /p:PortableBuild=false /p:EnableNgenOptimization=false ${passCI} \${buildIdOpts} \${stableOpts} \${packageOpts} /p:BinPlaceNETCoreAppPackage=true /p:OverridePackageSource=https:%2F%2Ftizen.myget.org/F/dotnet-core/api/v3/index.json ${authorsOpts}")
```

Comment:
Why is `EnableNgenOptimization` disabled?

Reply:
To use `EnableNgenOptimization` option in official build we need to [restore](https://github.com/dotnet/corefx/blob/2e6661d4cceeff2549b0299ec54e9f3b34c09b20/eng/pipelines/corefx-base.yml#L137) internal tools from authenticated channel, but we don't have credential for it.
```
$ ~/workspace/release/corefx_master_armel_Release/netcore-jenkins/dotnet-dev/dotnet restore --configfile eng/internal/NuGet.config --packages ./.packages eng/common/internal/Tools.csproj

Configuring...
--------------
A command is running to populate your local package cache to improve restore speed and enable offline access. This command takes up to one minute to complete and only runs once.
Decompressing 100% 6025 ms
Expanding 100% 5784 ms
  Restoring packages for /var/lib/jenkins/workspace/release/corefx_master_armel_Release/dotnet_corefx/eng/common/internal/Tools.csproj...
/var/lib/jenkins/workspace/release/corefx_master_armel_Release/netcore-jenkins/dotnet-dev/sdk/2.1.403/NuGet.targets(114,5): error : Unable to load the service index for source https://devdiv.pkgs.visualstudio.com/_packaging/dotnet-core-internal-tooling/nuget/v3/index.json. [/var/lib/jenkins/workspace/release/corefx_master_armel_Release/dotnet_corefx/eng/common/internal/Tools.csproj]
/var/lib/jenkins/workspace/release/corefx_master_armel_Release/netcore-jenkins/dotnet-dev/sdk/2.1.403/NuGet.targets(114,5): error :   Response status code does not indicate success: 401 (Unauthorized). [/var/lib/jenkins/workspace/release/corefx_master_armel_Release/dotnet_corefx/eng/common/internal/Tools.csproj]
```